webclient/core/views.py

```diff
@@ -930,7 +930,15 @@ class SearchListView(ExportMixin, LoginRequiredMixin, SingleTableMixin, FilterVi
     progress_bar_coefficient = 0.8
 
     def create_export(self, export_format):
-        def update_progress_bar(r_inner, key_inner, new_value):
+        from redis import Redis
+
+        def check_if_aborted(r_inner: Redis, key_inner: str):
+            aborted = r_inner.get(key_inner) == -1
+            if aborted:
+                r_inner.delete(key_inner)
+            return aborted
+
+        def update_progress_bar(r_inner: Redis, key_inner: str, new_value: int):
             logger.debug(
```

Comment:
Prosím smazat ten logger, při exportu jich může vyjet 100 a je zbytečné to uchovávat.

Reply:
Hotovo.